src/adapter/z-stack/adapter/backup.ts

```diff
@@ -181,7 +214,8 @@ async function Restore(znp: Znp, backupPath: string, options: NetworkOptions): P
 
     await znp.request(Subsystem.SYS, 'osalNvWrite', backup.data.ZCD_NV_EXTADDR);
     await znp.request(Subsystem.SYS, 'osalNvItemInit', ZCD_NV_NIB, null,
-        [ZnpCommandStatus.NV_ITEM_INITIALIZED]);
+        [ZnpCommandStatus.SUCCESS, ZnpCommandStatus.NV_ITEM_INITIALIZED]);
+    await znp.request(Subsystem.SYS, 'osalNvWrite', backup.data.ZCD_NV_NIB);
```

Comment:
Could you explain why this was added, besides that I have no comments. looks good :smile:

Reply:
This handle case, if NV item was already initialized, but we need to ovreride it.
zStack return `ZnpCommandStatus.SUCCESS` in this case.
We are overriding it with new value in line 218.

Reply:
Thanks for claryfying.